wwwapp/models.py

```diff
@@ -73,7 +114,7 @@ def __str__(self):
 
 class UserInfo(models.Model):
     """Info needed for camp, not for qualification."""
-    pesel = models.CharField(max_length=20, blank=True, default="")
+    pesel = PESELField(max_length=20, blank=True, default="")
```

Comment:
`max_length` could be adjusted to match actual PESEL length, explicitly specifying `min_length` may be a good idea too if it doesn't throw out empty values.

Reply:
@krzys-h `min_length` and `max_length` can't be used simultaneously. There is a workaround, but I think it's ok to leave the checks for the `validate` method, as I can handle the case of empty PESEL provided.